cp1616/CMakeLists.txt

```diff
@@ -23,6 +24,7 @@ catkin_package(
 )
 
 add_executable(cp1616_test ${PROJECT_SOURCE_DIR}/src/cp1616_main.cpp)
+target_link_libraries (Cp1616RosLib yaml-cpp)	
```

Comment:
Library names should be identical to the package name by convention.  Should also be lowercase.